### DISCUSSION TITLE
Readme needs more details

### DISCUSSION THREAD

**Discussion by ArmatorPL** [woj-i/degiro-pit#2] [Q&A] [answered]
Hi, I am not a coder and have no idea which tools to use to run your app. What are the system requirements? Cmd? Git Bash? etc.. An installation instruction would be very useful for people that are not coding. Thanks :)
![image](https://user-images.githubusercontent.com/36926807/111884389-b8612600-89c1-11eb-8822-d6507ea3b6a5.png)

**Comment by woj-i** [comment, marked as answer]
Thank you for your suggestion @ArmatorPL . I'll try to prepare more detailed documentation until the end of this week.

**Reply by ArmatorPL** [reply]
Cool, thanks for the quick reply :) Looking forward to test it. Cheers

**Reply by woj-i** [reply]
@ArmatorPL here it is https://github.com/woj-i/degiro-pit/blob/main/README.md
Looking forward for your feedback! Thanks!

**Reply by ArmatorPL** [reply]
Thanks :) will give it a try during the weekend and let you know my thoughts. Cheers
